packages/dds/tree/src/test/snapshots/opFormat.spec.ts

```diff
@@ -0,0 +1,75 @@
+import { AttachState } from "@fluidframework/container-definitions";
+import { type SessionId, createIdCompressor } from "@fluidframework/id-compressor";
+import {
+	type MockContainerRuntime,
+	MockContainerRuntimeFactory,
+	MockFluidDataStoreRuntime,
+	MockStorage,
+} from "@fluidframework/test-runtime-utils";
+import { type ITree, SchemaFactory } from "../../index.js";
+import { SharedTree } from "../../treeFactory.js";
+import { takeJsonSnapshot, useSnapshotDirectory } from "./snapshotTools.js";
+
+/**
+ * This suite provides some e2e snapshot coverage for how SharedTree ops look.
+ * Prefer to put exhaustive aspects of the op format in more specific snapshot tests.
```

Comment:
I'm not 100% convinced we want this as a snapshot test. As-is we actually do have snapshot coverage for basically all bits of the format. The upside of a test like this is it gives reasonable visibility into what we're actually putting over the wire (which might e.g. lead to better awareness/scrutinization/improvement). The downside is that the snapshot tests take dependencies on implementation details like the fact that initializing value schema at the root requires first doing a schema update to a superset schema, or details like how many allocated ids we make in the overall initialization flow.

I'd *guess* this test isn't super noisy (i.e. constantly needing to be updated for changes that don't affect persisted format), and that the benefits outweigh the overhead of the noise we do get, but curious how the reviewer feels.